### DISCUSSION TITLE
Add methods to estimate warm-up

### DISCUSSION THREAD

**Discussion by MichaelClerx** [pints-team/pints#1717] [Announcements]
E.g. Geweke

**Comment by MichaelClerx** [comment]
Could also be used to veto stopping criteria?

**Comment by ben18785** [comment]
Sounds interesting! What's the reference here? Sorry, there are a few things that come up with Geweke and warm-up and fear I'm missing the key reference.

**Comment by MichaelClerx** [comment]
This article mentions it: https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-017-0433-1

**Comment by ben18785** [comment]
Great. Thanks.

**Comment by MichaelClerx** [comment]
See also #1140
